lighthouse-core/audits/content-width.js

```diff
@@ -55,8 +55,8 @@ class ContentWidth extends Audit {
       return '';
     }
 
-    return 'The content scroll size is ' + artifact.scrollWidth + 'px, ' +
-        'whereas the viewport size is ' + artifact.viewportWidth + 'px.';
+    return 'The content scroll size is ' + artifact.innerWidth + 'px, ' +
+        'whereas the viewport size is ' + artifact.outerWidth + 'px.';
```

Comment:
Sorry. I'm still caught up in lingo and intended behavior. Does http://jsbin.com/murusonore/edit?html,console,output capture what you're after? 

`document.documentElement.scrollWidth == content scroll size`
`window.innerWidth == visible viewport size`

(In the bin example, both of these metrics are < `window.outerWidth` because it's running inside the iframe...which is smaller than the viewport.)

----
Other refs:

> `innerWidth`: Width (in pixels) of the browser window viewport including, if rendered, the vertical scrollbar.

https://developer.mozilla.org/en-US/docs/Web/API/Window/innerWidth

> `outerWidth` gets the width of the outside of the browser window. It represents the width of the whole browser window including sidebar (if expanded), window chrome and window resizing borders/handles. 

https://developer.mozilla.org/en-US/docs/Web/API/Window/outerWidth